traffic_ops/traffic_ops_golang/crconfig/handler.go

```diff
@@ -296,11 +294,5 @@ func SnapshotOldGUIHandler(w http.ResponseWriter, r *http.Request) {
 	}
 
 	api.CreateChangeLogRawTx(api.ApiChange, "Snapshot of CRConfig performed for "+cdn, inf.User, inf.Tx.Tx)
-	http.Redirect(w, r, "/tools/flash_and_close/"+url.PathEscape("Successfully wrote the CRConfig.json!"), http.StatusFound)
-}
-
-func writePerlHTMLErr(w http.ResponseWriter, r *http.Request, tx *sql.Tx, logErr error, err error) {
-	log.Errorln(logErr.Error())
-	tx.Rollback()
```

Comment:
Interesting. I haven't seen many transaction rollbacks. Is this something we still need to account for before redirecting?

Reply:
Normally the `api.HandleErr` function takes care of this for us.